src/test/java/Homework16.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 {
+@Test
+public static void registrationNavigation () {
+
+        //  Added ChromeOptions argument below to fix websocket error
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        WebElement registrationLink = driver.findElement(By.cssSelector("[id='hel']"));
+        registrationLink.click();
+
+        String registrationUrl = "https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
+        driver.quit();
+    }
+}
+
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226556705-d60518a4-7829-4a6d-926f-c6958e64e30d.png)

HINT

`public class Homework16 extends BaseTest`